az-digital/az_quickstart
# Add configuration for anonymous review

## Description
Adds configuration for Access Unpublished and updates user role permissions.

Review site: https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-837.probo.build/

### Changes from D7

In D7, you would normally retrieve the hashed link when viewing the node page after performing an update (it would appear in a notice dialog). In D9, you not retrieve the hashed link when editing the node; it is contained in a collapsible tab in the editing sidebar titled "Temporary unpublished access". This is [not documented in the module](https://www.drupal.org/project/access_unpublished/issues/2904716) anywhere, so we might want to provide some documentation for it on the new AZQS site (follow-up ticket).

The Drupal 7 version also had a custom UAQS module for the anonymous reviewer workflow. It doesn't look like that's necessary in D9 (just need Access Unpublished and core's Content Moderation).

### How to Enable & Test

1. Enable the "editorial" workflow on the "Page" content type
2. Enable "Access Unpublished" module
3. Edit/create a Page and save it as a "Draft"
4. Edit the page again and open the "Temporary unpublished access" tab in the editor sidebar
5. Enable a temporary access URL

It looks like you must first save a page as a Draft and save it before the access unpublished URL becomes an option in the editing interface. That's a little awkward (I would expect to be able to create a new draft and create a hashed URL for it in the same step) so we might want to consider documenting that.

## Related Issues
Closes #578
Follow up: #924 (documentation) 

## How Has This Been Tested?
Tested locally and on Probo.

## Types of changes

## Checklist:
